pkg/provider/provider.go

```diff
@@ -159,5 +160,17 @@ func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}
 		return nil, diags
 	}
 
+	if testing {
+		_, err := db.Exec("ALTER SYSTEM SET enable_managed_clusters = true;")
```

Comment:
If this is live in production, can we check if this can get removed from the testing connection?

Reply:
Managed clusters are enabled by default for all environments. The flag was removed in [Materialize #20339](https://github.com/MaterializeInc/materialize/pull/20399).

Reply:
Seems like the PR above is not yet merged so we still need that alter statement